packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts

```diff
@@ -202,6 +203,27 @@ export const updateProfileHandler = async ({ ctx, input }: UpdateProfileOptions)
     data.avatarUrl = null;
     data.avatar = null;
   }
+  if (input.completedOnboarding) {
+    const userTeams = await prisma.user.findFirst({
+      where: {
+        id: user.id,
+      },
+      select: {
+        teams: {
+          select: {
+            id: true,
+          },
+        },
+      },
+    });
+    if (userTeams && userTeams.teams.length > 0) {
+      await Promise.all(
+        userTeams.teams.map(async (team) => {
+          await updateNewTeamMemberEventTypes(user.id, team.id);
+        })
+      );
+    }
```

Comment:
updateProfile mutation is used in a lot of pages. why do we want to repeat this  updateNewTeamMemberEventTypes step everytime

Reply:
only doing it when if `input.completedOnboarding` is true
which is called only once i think.
this will only run once the user has completed onboarding
couldn't think of a better place to do it

Reply:
Oh okay. btw we already use updateNewTeamMemberEventTypes in acceptOrLeaveHandler.ts so whenever user accept invite. was that not working properly?